hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.test.tsx

```diff
@@ -34,13 +35,48 @@ describe("Inventory Page", () => {
             .calledWith(categoriesUri)
             .mockResolvedValue(categoryApiResponse);
 
-        const { getByText } = render(<Inventory />);
+        const { getByText, queryByText } = render(<Inventory />);
 
         await waitFor(() => {
             expect(getByText(mockHardware[0].name)).toBeInTheDocument();
             expect(getByText(mockCategories[0].name)).toBeInTheDocument();
         });
 
+        // by default the hardwareUri has a limit of 100
```

Comment:
This test doesn't really make sense

- `mockHardware` is always going to be less than 100 elements. It doesn't change as part of the test, and we're never going to add that many things to it
- "100" is not hard coded anywhere in the front end, and is not a special number. We could just as easily change the backend to use 50 or 200 items per page (to test this PR, I set it to 1 item per page).

This test is passing because `mockHardware` happens to be less than 100 elements, and the count generated by `makeMockApiListResponse` is the length of the data you pass in (ie, "load more" will never appear). If we put 101 elements in that mock data array, this test would fail.

I would remove this block entirely, and let the functionality be covered by your new test cases

Reply:
I don't understand how it would fail because there is an else block but I agree, I will test the load more button in separate tests.

Reply:
If we added a bunch more elements to `mockHardware`, we'd go into the else block, and the test would fail because count === hardware.length, so the "load more" button won't be there

Reply:
oh I right, we are mocking the endpoint so there is no automatic limit 🤦‍♀️